README.md

```diff
@@ -27,7 +27,106 @@
 
 Hazelcast supports EC2 auto-discovery. It is useful when you do not want to provide or you cannot provide the list of possible IP addresses. 
 
-To configure your cluster to use EC2 auto-discovery, follow the below steps:
+There are two possible ways to configure your cluster to use EC2 auto-discovery.
+You can either choose to configure your cluster with `AwsConfig` (or `aws` element in your XML config)
+or you can choose configuring AWS discovery using [Discovery SPI](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#discovery-spi) implementation.
+Starting with hazelcast-aws-1.2, it's strongly recommended to use Discovery SPI implementation as
+the former one will be deprecated.
+
+###Configuring AWS Discovery using Discovery SPI for Hazelcast Cluster Members
+
+- Add the *hazelcast-aws.jar* dependency to your project. Note that it is also bundled inside *hazelcast-all.jar*. The Hazelcast aws module does not depend on any other third party modules.
```

Comment:
I think module name is used as "hazelcast-aws" throughout the README. So for the sake of consistency "The Hazelcast aws module ...." can be changed as "The hazelcast-aws module..."